functional-samples/tutorial.tabs-manager/README.md

```diff
@@ -0,0 +1,209 @@
+# Manage tabs
+
+This [sample](https://developer.chrome.com/docs/extensions/get-started/tutorial/popup-tabs-manager) build your first tabs manager.
+
+## Overview
+
+The guide outlines how to build a tabs manager to organize your Chrome extension and Chrome Web Store documentation tabs.
+
+<div align="center">
+  <figure>
+  <img src="https://developer.chrome.com/static/docs/extensions/get-started/tutorial/popup-tabs-manager/image/tabs-manager-extension-po-725ac725aaa4d_1440.png" width="auto" height="auto">
+  <figcaption>Tabs Manager extension</figcaption>
+</figure>
+</div>
+<br><br>
+
+In this guide, we're going to explain how to do the following:
+
+- Create an extension popup using the Action [API](https://developer.chrome.com/docs/extensions/reference/api/action).
+- Query for specific tabs using the Tabs [API](https://developer.chrome.com/docs/extensions/reference/api/tabs).
+- Preserve user privacy through narrow host permissions.
+- Change the focus of the tab.
+- Move tabs to the same window and group them.
+- Rename tab groups using the TabGroups [API](https://developer.chrome.com/docs/extensions/reference/api/tabGroups).
+
+## Implementation Notes
+
+### Step 1: Add the extension data and icons
```

Comment:
This section is intended for a very brief summary of any specific parts of this sample that might be surprising or unintuitive. Could you remove the tutorial content here and add anything you think is relevant, or otherwise simply remove the section?

Reply:
@abrehan2, could you also take a look at this?